python/opae.admin/setup.py

```diff
@@ -39,7 +39,8 @@
             'fpgaotsu = opae.admin.tools.fpgaotsu:main',
             'fpgaport = opae.admin.tools.fpgaport:main',
             'bitstreaminfo = opae.admin.tools.bitstream_info:main',
-            'opaevfio = opae.admin.tools.opaevfio:main'
+            'opaevfio = opae.admin.tools.opaevfio:main',
+            'regmap-debugfs = opae.admin.tools.regmap_debugfs:main'
```

Comment:
You could add comma at the end of this line. Next time, if any addition will need to be done to this file, it will be only marked as "added" instead of "removed and added".

Reply:
If it is okay to end with a comma, I'll change it, because it is better.